base_geoengine/static/src/js/views/form_renderer.js

```diff
@@ -0,0 +1,19 @@
+odoo.define('base_geoengine.form_renderer', function (require) {
+    "use strict";
+
+    var FormRenderer = require('web.FormRenderer');
+
+    FormRenderer.include({
+        // _updateView refreshes the view after a pager action.
+        // Add the TabListener to geo_type widget so that _renderMap is called
```

Comment:
We can probably keep `_renderMap` in this comment even if not completely accurate due to last change.